src/rust/engine/src/externs.rs

```diff
@@ -101,6 +105,12 @@ impl Externs {
   pub fn create_exception(&self, msg: String) -> Value {
     (self.create_exception)(self.context, msg.as_ptr(), msg.len() as u64)
   }
+
+  pub fn invoke_runnable(&self, runnable: &Runnable) -> RunnableComplete {
+    let args_clone: Vec<Value> = runnable.args().clone();
+    (self.invoke_runnable)(
```

Comment:
Yes: to differentiate between the struct field and the function of the same name.